src/main/java/com/hubspot/jinjava/lib/filter/SliceFilter.java

```diff
@@ -79,21 +83,32 @@ public Object filter(Object var, JinjavaInterpreter interpreter, String... args)
         0,
         args[0]
       );
+    } else if (slices > MAX_SLICES) {
+      ENGINE_LOG.warn(
```

Comment:
This won't be viewable by developers. We'd need to add a warning to the context like https://github.com/HubSpot/jinjava/blob/0de0e078e14e210719c622a6032621bf5c5e49af/src/main/java/com/hubspot/jinjava/lib/filter/PlusTimeFilter.java#L64

Reply:
replaced by your suggestion